### PR TITLE
Update additive.py

### PR DESCRIPTION
**Pull request by ErwanQuintin**
Added simple black body model. The norm is kept dimensionless, of order ~1e-5, similar to what is in Xspec. Might need to change it in future updates, to be consistent with other standardised normalisations.

### CODE REVIEW

File: src/jaxspec/model/additive.py [hunk 1]
```diff
@@ -74,4 +74,27 @@ def __call__(self, energy):
...
+        :math:`K` : :math:`L_{39}/D_{10}^{2}`, where :math:`L_{39}` is the source luminosity in units of `1O^{39}` erg/s and `D_{10}` is the distance to the source in units of 10 kpc
+         :math:`\left[\right]`
+    """
+
+    def __call__(self, energy):
+
+        kT = hk.get_parameter('kT', [], init=Constant(11/3))
+        norm = hk.get_parameter('norm', [], init=Constant(1))
+
+        return norm*8.0525*energy**2/((kT**4)*(jnp.exp(energy/kT)-1))
```
Comment (renecotyfanboy) [inline src/jaxspec/model/additive.py:100]: I suppose 8.0525 is the conversion factor between adimensioned norm and spectrum units

**Review by renecotyfanboy — APPROVED** [review]
